cosmpy/ibc/core/connection/rest_client.py

```diff
@@ -54,7 +54,7 @@ def Connection(self, request: QueryConnectionRequest) -> QueryConnectionResponse
 
         :param request: QueryConnectionRequest
         :return: QueryConnectionResponse
-        """
+        """  # noqa: D401
```

Comment:
I don't see any `raises` missing.

Reply:
Similarly below

Reply:
D401: First line should be in imperative mood; try rephrasing

Reply:
```
cosmpy/common/rest_client.py:127:1: D401 First line should be in imperative mood; try rephrasing
cosmpy/common/utils.py:30:1: D401 First line should be in imperative mood; try rephrasing
cosmpy/crypto/address.py:90:1: D401 First line should be in imperative mood; try rephrasing
cosmpy/ibc/core/connection/rest_client.py:52:1: D401 First line should be in imperative mood
cosmpy/ibc/core/connection/rest_client.py:64:1: D401 First line should be in imperative mood
cosmpy/ibc/core/connection/interface.py:42:1: D401 First line should be in imperative mood
cosmpy/ibc/core/connection/interface.py:51:1: D401 First line should be in imperative mood
```

I didn't find any strong solution for this.

Reply:
```
cosmpy/common/rest_client.py:127:1: D401 First line should be in imperative mood; try rephrasing
cosmpy/common/utils.py:30:1: D401 First line should be in imperative mood; try rephrasing
cosmpy/crypto/address.py:90:1: D401 First line should be in imperative mood; try rephrasing
cosmpy/ibc/core/connection/rest_client.py:52:1: D401 First line should be in imperative mood
cosmpy/ibc/core/connection/rest_client.py:64:1: D401 First line should be in imperative mood
cosmpy/ibc/core/connection/interface.py:42:1: D401 First line should be in imperative mood
cosmpy/ibc/core/connection/interface.py:51:1: D401 First line should be in imperative mood
```

I didn't find any strong solution for this.